meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -0,0 +1,231 @@
+import org.apache.helix.metaclient.api.ConnectStateChangeListener;
+import org.apache.helix.metaclient.api.DataChangeListener;
+import org.apache.helix.metaclient.api.DataUpdater;
+import org.apache.helix.metaclient.api.DirectChildrenChangeListener;
+import org.apache.helix.metaclient.api.DirectEntrySubscribeResult;
+import org.apache.helix.metaclient.api.MetaClientInterface;
+import org.apache.helix.metaclient.api.OpResult;
+import org.apache.helix.metaclient.api.PersistSubEntryChangeListener;
+import org.apache.helix.zookeeper.api.client.RealmAwareZkClient;
+
+
+public class ZkMetaClient implements MetaClientInterface {
+
+  private RealmAwareZkClient _zkClient;
+
+  public ZkMetaClient() {
+
+  }
+
+  @Override
+  public void create(String key, Object data, EntryMode mode) {
+
+  }
+
+  @Override
+  public void create(String key, Object data, EntryMode mode, long ttl) {
+
+  }
+
+  @Override
+  public void set(String key, Object data, int version) {
+
+  }
+
+  @Override
+  public Object update(String key, DataUpdater updater) {
+    return null;
+  }
+
+  @Override
+  public Stat exists(String key) {
+    return null;
+  }
+
+  @Override
+  public Object get(String key) {
+    return null;
+  }
+
+  @Override
+  public List<String> getSubEntryKeys(String path) {
+    return null;
+  }
+
+  @Override
+  public int countSubEntries(String path) {
+    return 0;
+  }
+
+  @Override
+  public boolean delete(String path) {
+    return false;
+  }
+
+  @Override
+  public boolean recursiveDelete(String path) {
+    return false;
+  }
+
+  @Override
+  public void asyncCreate(String key, Object data, int version, long ttl,
+      AsyncCallback.VoidCallback cb) {
+
+  }
+
+  @Override
+  public void asyncSet(String key, Object data, int version, AsyncCallback.VoidCallback cb) {
+
+  }
+
+  @Override
+  public void asyncUpdate(String key, DataUpdater updater, AsyncCallback.VoidCallback cb) {
+
+  }
+
+  @Override
+  public void asyncGet(String key, AsyncCallback.DataCallback cb) {
+
+  }
+
+  @Override
+  public void asyncCountSubEntries(String path, AsyncCallback.DataCallback cb) {
+
+  }
+
+  @Override
+  public void asyncExist(String key, AsyncCallback.StatCallback cb) {
+
+  }
+
+  @Override
+  public void asyncDelete(String keys, AsyncCallback.VoidCallback cb) {
+
+  }
+
+  @Override
+  public void asyncTransaction(String keys, AsyncCallback.TransactionCallback cb) {
+
+  }
+
+  @Override
+  public ConnectState connect() {
+    return null;
+  }
+
+  @Override
+  public void disconnect() {
+
+  }
+
+  @Override
+  public boolean subscribeDataChange(String key, DataChangeListener listener,
+      DataChangeListener.ChangeType eventType, boolean skipWatchingNonExistNode,
+      boolean persistListener) {
+    return false;
+  }
+
+  @Override
+  public DirectEntrySubscribeResult subscribeDirectEntryChange(String key,
+      DirectChildrenChangeListener listener, boolean skipWatchingNonExistNode,
```

Comment:
Why is the naming here not consistent? Shouldn't it all be `DirectChildrenChange`?